eclass/java-pkg-simple.eclass

```diff
@@ -453,18 +565,42 @@ java-pkg-simple_src_compile() {
 			|| die "updating MANIFEST.MF failed"
 		rm -f "${T}/add-to-MANIFEST.MF" || die "cannot remove"
 	fi
+	JAVA_INTERNAL_JAR_FILES+=( "${PWD}/${JAVA_JAR_FILENAME}" )
```

Comment:
As I learned, there is a use case where a jar-file is generated but should not be installed. So we need a way to tell `java-pkg-simple_src_compile` to not automatically add to `JAVA_INTERNAL_JAR_FILES`.

Reply:
No, no general support. Just that it stores which jars where compiled and installs all of them in `java-pkg-simple_src_install`. But I can save this part for a later PR.

Reply:
I have dropped the changes for auto-installing jars. `java-pkg-simple_src_install` is still touched to install the accumulated sources from the `java-pkg-simple_compile_jar` runs. Are you fine with this?

Reply:
I reduced the changes further to keep it simple. `java-pkg-simple_src_install` is not touched anymore.